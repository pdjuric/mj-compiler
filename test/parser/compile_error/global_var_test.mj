program main_test

const int c_a;
int v_a;
class A {}

int A;
int ord;
int this;
char v_a, c_a;

{
    void main() { }
}
