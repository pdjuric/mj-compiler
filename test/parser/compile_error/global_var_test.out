ERROR 17:06:41,702 - line   7 - Identifier A already defined in this scope.
ERROR 17:06:41,702 - line   9 - "this" is not an allowed variable name.
ERROR 17:06:41,703 - line  10 - Identifier v_a already defined in this scope.
ERROR 17:06:41,703 - line  10 - Identifier c_a already defined in this scope.